Dockerfile

```diff
@@ -118,8 +118,9 @@ RUN apk --update --no-cache add -t build-dependencies \
   && sed -i '/runningUser/d' lnms \
   && echo "foreach (glob(\"/data/config/*.php\") as \$filename) include \$filename;" >> config.php \
   && echo "foreach (glob(\"${LIBRENMS_PATH}/config.d/*.php\") as \$filename) include \$filename;" >> config.php \
-  && git clone https://github.com/librenms-plugins/Weathermap.git ./html/plugins/Weathermap \
   && chown -R nobody.nogroup ${LIBRENMS_PATH} \
+  && git clone https://github.com/librenms-plugins/Weathermap.git ./html/plugins/Weathermap \
+  && chown -R librenms:librenms ./html/plugins/Weathermap \
```

Comment:
I suggest clone it where it is, then change the owner to librenms:librenms for these required directories only:
./html/plugins/Weathermap/configs
./html/plugins/Weathermap/output